lib/reportData.js

```diff
@@ -225,6 +225,14 @@ ReportData = function (baseURL, mp) {
     };
 
     /**
+     * @instance
```

Comment:
This should have a better docBlock on it. Also, 'queue' is an array of query param objects, not URLs: "Object literal containing key value pairs to encode as query string params on the end of the url." The var 'query' should be renamed to have a more descriptive name, like 'eventParamQueue.'